client/src/components/ResultItem/ResultItem.js

```diff
@@ -0,0 +1,34 @@
+import './ResultItem.css';
+
+import { Highlight } from 'react-instantsearch-dom';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/* Algolia's Hit: display a result item from search */
+function ResultItem(props) {
+  // a hit (result) object that contains all info about a result item
+  const { hit } = props;
+
+  // Open url in a new tab
+  const handleLinkClicked = () => {
+    const url = `https://www.${hit.data.link}`;
+    const win = window.open(url, '_blank');
+    win.focus();
+  };
+  return (
+    <article>
+      <h1 onClick={handleLinkClicked} className="highlight-header">
```

Comment:
Easier (and more accessible) to just use a styled \<a\> element here.  Then you can do

target="_blank"
rel="noopener noreferrer"

which will avoid security issues that Safari loves to complain about